toolchain/check/context.h

```diff
@@ -208,22 +212,32 @@ class Context {
   auto AddNameToLookup(SemIR::NameId name_id, SemIR::InstId target_id) -> void;
 
   // Performs name lookup in a specified scope for a name appearing in a
-  // declaration, returning the referenced instruction. If scope_id is invalid,
-  // uses the current contextual scope.
+  // declaration. If found, returns the referenced instruction and false. If
+  // poisoned, returns invalid instructions and true. poisoned. If scope_id is
```

Comment:
Just a couple typos:

```suggestion
  // poisoned, returns an invalid instruction and true. If scope_id is
```

Might also want to rewrap the comment after the fix.

Reply:
Done.